.github/workflows/test.yaml

```diff
@@ -24,17 +24,22 @@ jobs:
         python-version: ${{ matrix.python-version }}
 
     - name: Install Packages
-      run: pip install -r requirements_dev.txt
+      run: |
+        pip install -r requirements.txt
+        pip install -r test-requirements.txt
 
     - name: Test
       env:
         BW_ACCOUNT_ID: ${{ secrets.BW_ACCOUNT_ID }}
         BW_USERNAME: ${{ secrets.BW_USERNAME }}
         BW_PASSWORD: ${{ secrets.BW_PASSWORD }}
+        BW_USERNAME_FORBIDDEN: ${{ secrets.BW_USERNAME_FORBIDDEN }}
```

Comment:
May be rename when you get a chance again to `BW_FORBIDDEN_USERNAME`

Reply:
We could? Id prefer `BW_USERNAME_{attribute}` though because then they're all in order on the secrets page and it makes it easier to find one if we need to change it

Reply:
Yeah, I like `BW_FORBIDDEN_USERNAME` for being alphabetical but I think AJ's argument for `BW_USERNAME_{attribute}` makes sense.

Reply:
On that note we might want to rename all of the phone numbers to use `PHONE_NUMBER_{attribute} since we could have a lot - especially with @matthewkmartin's manteca stuff

Reply:
👍  Sorting benefit makes sense, may be refactor later where we can benefit from both!